embed/config.go

```diff
@@ -299,6 +299,14 @@ func (cfg *Config) Validate() error {
 	if err := checkBindURLs(cfg.ListenMetricsUrls); err != nil {
 		return err
 	}
+	if err := checkHostURLs(cfg.APUrls); err != nil {
+		// TODO: return err in v3.4
+		plog.Warning(err)
```

Comment:
should these say something like "advertise-peer-urls %q is deprecated (%v)" if the intent is to eventually exit out? warning with only the error doesn't seem clear enough